releases/goerli/1.9.6/index.html

```diff
@@ -10,7 +10,7 @@ <h2>
 
 			<ul>
 				<li><a href="/releases/goerli/1.9.6/contracts.json">Contract addresses</a></li>
-				<li><a href="https://ipfs.io/ipfs/QmTCS3ezDGrHRtrfjY1aSLeHZnJgxzSwn1NScrdNDfQGbY">Contract addresses (IPFS hosted)</a></li>
+				<li><a href="https://ipfs.io/ipfs/QmNUWpfaWit5m8ATxN3c74towREeA2x36P495oR9ksdcyJ">Contract addresses (IPFS hosted)</a></li>
```

Comment:
`SyntaxError: JSON.parse: expected double-quoted property name at line 261 column 1 of the JSON data`

Reply:
fixed